wgpu-hal/src/gles/command.rs

```diff
@@ -421,10 +421,7 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
             }
         }
 
-        // set the draw buffers and states
-        self.cmd_buffer
```

Comment:
I don't think we can do this, since the draw targets are required for the clears on the normal path